assets/js/load-chart.js

```diff
@@ -38,7 +38,8 @@ function parseDateString (date) {
 function toolTip (date, wpt_sha, servo_version, score, engine) {
     return `
         <b>${formatDate(date)}</b></br>
-        Score: <b>${score / 10}</b></br>
+        Score: <b>${score.per_mille / 10}</b></br>
```

Comment:
Perhaps we could add a literal `%` symbol for "score" here too, even though it didn't originally have one.

Reply:
Done.